ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/TerminalPowBlockMonitor.java

```diff
@@ -267,4 +265,32 @@ private void onTerminalPowBlockFound(Bytes32 blockHash) {
   private boolean notYetFound(Bytes32 blockHash) {
     return !foundTerminalBlockHash.map(blockHash::equals).orElse(false);
   }
+
+  private void callExchangeTransitionConfiguration(SpecConfigBellatrix specConfigBellatrix) {
```

Comment:
I think the idea is that we keep polling this to ensure the config remains in sync and the EL is still available.  Probably makes sense to have the same life cycle as searching for the TTD block - once the merge is completed successfully block import will fail if the configs are different so it won't go unnoticed and we can stop doing this check as well.

Though I think we might want to start doing this checks before the merge fork actually activates which is earlier than we'd be running the `TerminalPowBlockMonitor`.